cli/src/athenapdf.js

```diff
@@ -46,6 +46,7 @@ athena
     .option("--no-cache", "disables caching")
     .option("--ignore-certificate-errors", "ignores certificate errors")
     .option("--ignore-gpu-blacklist", "Enables GPU in Docker environment")
+    .option("--disable-http-cache", "Disables Electron disk cache for HTTP requests.")
```

Comment:
I agree that it would be better to make disable electron cache with `--no-cache` option instead of introducing a new one.